multicluster/cmd/check.go

```diff
@@ -96,6 +96,10 @@ non-zero exit code.`,
 	}
 	cmd.Flags().StringVarP(&options.output, "output", "o", options.output, "Output format. One of: basic, json")
 	cmd.Flags().DurationVar(&options.wait, "wait", options.wait, "Maximum allowed time for all tests to pass")
+	cmd.Flags().Bool("proxy", false, "")
+	cmd.Flags().MarkHidden("proxy")
+	cmd.Flags().StringP("namespace", "n", "", "")
+	cmd.Flags().MarkHidden("namespace")
```

Comment:
Maybe make a follow-up issue for supporting these?

Reply:
I don't think there are really any data-plane checks that the multicluster extension has to do.